src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -215,65 +179,62 @@ namespace System.Runtime.CompilerServices
         verifier.VerifyDiagnostics();
     }
 
+    /// <summary>
+    /// File-local InterceptsLocationAttribute from another compilation is not considered to *duplicate* an interception, even if it is inherited.
+    /// See also <see cref="DuplicateLocation_03"/>.
+    /// </summary>
     [Fact]
     public void FileLocalAttributeDefinitions_02()
     {
-        var source0 = """
+        var source1 = """
```

Comment:
This is essentially testing something completely different than before the PR. The scenario just came to my mind while working on it and rather than moving to test plan and doing it later I decided to get it done now.